2013/hmda-macro.json

```diff
@@ -1458,7 +1458,7 @@
                     "values": ["1", "2", "3", "4", "5"]
                 },
                 {
-                    "label": "% Total Loans where action taken = 1-5",
+                    "label": "% of Total Loans where action taken = 1-5",
```

Comment:
You've capital cased all the other "Action Taken", why not here? (same for 2014)